maquetado/scss/_tech.scss

```diff
@@ -0,0 +1,52 @@
+.technology {
+  align-items: center;
+  background-color: $green;
+  border-radius: $tech-border-radius;
+  display: flex;
+  flex-direction: column;
+  margin: 20px;
+  max-width: 100%;
+  padding: 15px;
+  position: relative;
+  width: 205px;
+
+  &:before {
+    border: 1px solid $green;
+    border-radius: $tech-border-radius;
+    content: '';
+    height: 100%;
+    left: -10px;
+    position: absolute;
+    top: -10px;
+    transition: left 0.5s ease,
+                top 0.5s ease;
+    width: 100%;
+    z-index: -1;
+  }
+
+  &:hover:before {
+    left: 10px;
```

Comment:
I recommend you this [article](https://medium.com/outsystems-experts/how-to-achieve-60-fps-animations-with-css3-db7b98610108), you should avoid use the properties that trigger the `painting`, you can use the `transform` property

Reply:
Check this please. The `transform` way to do the animation is the better way